sites/laserfocusworld/server/templates/index.marko

```diff
@@ -81,7 +81,7 @@ $ const adSlots = {
       <endeavor-content-query-section-list
         limit=3
         section-alias="software-accessories"
-        header={ title: "Software Accessories", href: "/software-accessories" }
+        header={ title: "Software & Accessories", href: "/software-accessories" }
```

Comment:
@brandonbk In the rendered HTML, is this output as `Software & Accessories` or `Software &amp; Accessories`? It should be the latter.